app/models/submission/judge_data.rb

```diff
@@ -106,7 +106,8 @@ def judgement
         case status
         when :timeout; "Time Limit Exceeded"
         when :walltime; "Time Limit Exceeded (Wall)"
-        when :runtime; "Runtime Error"
+        # Report runtime errors as complie errors when executing the compile command
+        when :runtime; "Compiler Error"
```

Comment:
This doesn't work as it will also change the verdicts on the submission page.
![image](https://user-images.githubusercontent.com/5953685/73044844-66616500-3ed0-11ea-9d25-00729d038709.png)